core/templates/dev/head/pages/exploration_editor/translation_tab/audio_translation_bar_directive.html

```diff
@@ -180,6 +195,11 @@
     <strong>Warning: </strong>Don't navigate to other tabs of this page before saving recorded audio, otherwise the recorded audio will be lost.
   </span>
 </div>
+<div class="oppia-translation-bottom-right-container" ng-if="!showRecorderWarning && !audioBlob ">
+  <span>
+    <strong>ProTip: </strong>Use key R to start/stop recording
```

Comment:
Make it `Use key "R" to start/stop recording.`

Reply:
Done